pkg/cmd/server/api/types.go

```diff
@@ -1202,3 +1202,13 @@ type ServiceServingCert struct {
 	// If this value is nil, then certs are not signed automatically.
 	Signer *CertInfo
 }
+
+// AdmissionPluginActivation can be used to enable to disable various admission plugins.
+// When this type is present as the `configuration` object under `pluginConfig` and *if* the admission plugin supports it,
+// this will cause an off by default admission plugin to be enabled
+type AdmissionPluginActivation struct {
```

Comment:
> I don't like that it's called pluginConfig (for starters).

It's inside of a stanza called `admissionConfig`, so if we changed the name it would be `admissionConfig.admissionConfig` which is probably worse than the current `admissionConfig.pluginConfig`.